packages/node/examples/express/package.json

```diff
@@ -0,0 +1,18 @@
+{
```

Comment:
Does/should this `packages/node/examples` dir need to be excluded from NPM publishes in `.npmignore`?

Reply:
https://github.com/readmeio/metrics-sdks/pull/418/commits/6070427634c4c28134254d2ffa525112e3c91146